### PR TITLE
[DependencyInjection] Update definitions.rst

### PR DESCRIPTION
**Pull request by mostefamed**
Rename the returned variable from setArgument method and the assigned variable to addArgument method

### CODE REVIEW

File: service_container/definitions.rst [hunk 1]
```diff
@@ -92,10 +92,10 @@ fetched from the container::
     
     // adds a new argument with the name of the argument
     // $argumentName = the name of the argument in the constructor
-    $argument = $definition->setArgument('$argumentName', $argumentValue);
+    $definition = $definition->setArgument('$argumentName', $argumentValue);
```
Comment (OskarStark) [inline service_container/definitions.rst:95]:
```suggestion
    $definition = $definition->setArgument($argumentName, $argumentValue);
```
?

Comment (maxhelias) [reply service_container/definitions.rst:95]: That's okay. It's to bind a variable on the definition

Comment (maxhelias) [reply service_container/definitions.rst:95]: However, I don't think we need to keep the return. It's confusing between `$definition` & `$argument` which shouldn't be the same "value" for the example.
And we should define an example that can be a real value for `$argumentValue` (new reference or parameter)

**Comment by javiereguiluz** [comment]
@mostefamed thanks and congrats on your first Symfony Docs contribution 🎉

**Pull request merged**